tutorials/scripting/gdextension/gdextension_cpp_example.rst

```diff
@@ -364,7 +364,8 @@ loaded for each platform and the entry function for the module. It is called ``g
     [configuration]
 
     entry_symbol = "example_library_init"
-    compatibility_minimum = "4.1"
+    compatibility_minimum = "4.2"
```

Comment:
Is there a reason to bump this? I would rather have people extend their compatibility ranges than chasing latest. Maybe the trade-off could be elaborated in the text.